Sources/TuistLoader/Models+ManifestMappers/ResourceFileElement+ManifestMapper.swift

```diff
@@ -24,40 +24,23 @@ extension XcodeGraph.ResourceFileElement {
                 excluded.formUnion(globs)
             }
 
+            if !path.pathString.isGlobComponent {
+                return [path]
+            }
+
             let files = try FileHandler.shared
                 .throwingGlob(.root, glob: String(path.pathString.dropFirst()))
                 .filter { !$0.isInOpaqueDirectory }
                 .filter(includeFiles)
                 .filter { !excluded.contains($0) }
 
-            if files.isEmpty {
-                if FileHandler.shared.isFolder(path) {
-                    logger.warning("'\(path.pathString)' is a directory, try using: '\(path.pathString)/**' to list its files")
-                } else {
-                    // FIXME: This should be done in a linter.
-                    logger.warning("No files found at: \(path.pathString)")
-                }
+            if files.isEmpty && FileHandler.shared.isFolder(path) {
```

Comment:
Won't `FileHandler.shared.isFolder(path)` always yield `false` here assuming that `path` is a glob at this point?

Reply:
Hi @pepicrft Thanks for you review, I'm a bit unsure here can you pls. suggest a fix for this 

cc: @vijaytholpadi

Reply:
correct! I've fixed the conditions 👍🏻